app/views/candidates/schools/_phase1.html.erb

```diff
@@ -101,12 +101,17 @@
       </div>
 
       <div id="school-availability-info" class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          Placement availability
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= format_school_availability(@school) %>
-        </dd>
+
+        <%- if @school.fixed_dates? -%>
+          <%= render partial: 'placement_dates', locals: { school: @school } %>
+        <%- else -%>
+          <dt class="govuk-summary-list__key">
+            Placement availability
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= format_school_availability(@school) %>
```

Comment:
this shows N/A if the school_profile use flexible dates as this method calls`Bookings::School#availability_info` rather than `Schools::SchoolProfile#availability_description.description`. Not an issue as we'll be writing to that column when the school publishes their school profile, just flagging this as it took me a minute to work out why I was seeing n/a even though my profile had flexible dates